infra/packer/testground-base.json

```diff
@@ -0,0 +1,84 @@
+{
+  "variables": {
+    "aws_access_key": "",
+    "aws_secret_key": ""
+  },
+  "builders": [{
+    "type": "amazon-ebs",
+    "access_key": "{{user `aws_access_key`}}",
+    "secret_key": "{{user `aws_secret_key`}}",
+    "region": "us-west-2",
+    "source_ami_filter": {
+      "filters": {
+        "virtualization-type": "hvm",
+        "name": "ubuntu/images/hvm-ssd/ubuntu-bionic-18.04-amd64-server-20191002",
+        "root-device-type": "ebs"
+      },
+      "owners": ["099720109477"],
+      "most_recent": true
+    },
+    "instance_type": "t2.micro",
+    "ssh_username": "ubuntu",
+    "ami_name": "testground-base {{timestamp}}"
+  }],
+  "provisioners": [
+    {
+      "type": "file",
+      "source": "filebeat.yml",
+      "destination": "/tmp/filebeat.yml"
+    },
+    {
+      "type": "file",
+      "source": "tweak-ansible.sh",
+      "destination": "/tmp/tweak-ansible.sh"
+    },
+    {
+      "type": "shell",
+      "inline": [
+        "sleep 30",
+        "sudo apt-get update",
+        "sudo apt-get install -y redis-server",
+        "sudo systemctl disable redis-server",
+        "sudo apt-get install -y build-essential",
+        "wget https://dl.google.com/go/go1.13.1.linux-amd64.tar.gz",
+        "sudo tar -C /usr/local -xzf go1.13.1.linux-amd64.tar.gz",
+        "rm go1.13.1.linux-amd64.tar.gz",
+        "echo 'export PATH=$PATH:/usr/local/go/bin' | sudo tee -a /etc/profile",
+        "sudo apt-get install -y apt-transport-https gnupg-agent",
+        "curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -",
+        "sudo add-apt-repository \"deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable\"",
+        "sudo apt-get update",
+        "sudo apt-get install -y docker-ce docker-ce-cli containerd.io",
+        "sudo usermod -aG docker ubuntu",
+        "wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -",
+        "echo 'deb https://artifacts.elastic.co/packages/7.x/apt stable main' | sudo tee -a /etc/apt/sources.list.d/elastic-7.x.list",
+        "sudo apt-get update",
+        "sudo apt-get install -y filebeat",
+        "sudo mv /tmp/filebeat.yml /etc/filebeat/filebeat.yml",
+        "sudo chown root. /etc/filebeat/filebeat.yml",
+        "sudo update-rc.d filebeat defaults 95 10",
+        "sudo apt-add-repository --yes --update ppa:ansible/ansible",
+        "sudo apt-get install -y ansible python3-pip python-pip",
+        "sudo bash /tmp/tweak-ansible.sh",
+        "sudo pip install boto boto3 botocore docker",
+        "sudo pip3 install boto boto3 botocore docker",
+        "sudo pip3 install awscli",
+        "sudo snap install ripgrep --classic",
+        "sudo apt-get install -y jq",
+        "sudo apt-get install -y nload",
+        "sudo apt-get install -y mosh",
```

Comment:
Can we install all packages in one command? Ideally, we'd just have a variable with a list of packages and then _one_ `sudo apt-get install -y ...`.